src/test/java/policies/RestrictAccessLocationTest.java

```diff
@@ -21,7 +21,7 @@ public class RestrictAccessLocationTest extends AbstracPolicyTest {
 	public void createRestrictAccessLocationAgreementTest() throws IOException {
 		ContractAgreement ca = createAgreement();
 		assertNotNull(ca);
-//		System.out.println(serializer.serialize(ca));
+		System.out.println(serializer.serialize(ca));
```

Comment:
Please do not do System.out in tests.
If required use logger instead.
(not sure if this syso is needed here)